.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "document-graph"]
+	path = document-graph
+	url = git@github.com:hypha-dao/document-graph.git
```

Comment:
Instead of using gitmodules, I'm using cmake to fetch the dependencies.

https://github.com/hypha-dao/voice-token/blob/master/src/CMakeLists-ExternalProjects.txt#L3-L7

We can remove the `gitsubmodules`.

Reply:
I will try and make cMake work

Reply:
cmake is now enabled and working

it's a lot slower than compile. but it works.